.baseline/copyright/999_palantir.txt

```diff
@@ -0,0 +1 @@
+(c) Copyright ${today.year} Palantir Technologies Inc. All rights reserved.
```

Comment:
can you delete this? otherwise this copyright will be used in the default intellij setup. the excavator OSS check removes this file after doing the normal baseline upgrade.

Reply:
https://github.com/palantir/safe-logging/pull/33